eclipse-collections/src/main/java/org/eclipse/collections/impl/list/mutable/MultiReaderFastList.java

```diff
@@ -624,34 +624,6 @@ public int lastIndexOf(Object o)
         }
     }
 
-    @Override
```

Comment:
I think these should be default methods in `MultiReaderList`.

Reply:
Same comment. Presumably MultiReaderSet too?

Reply:
Yes